test/1d_quadratures.cxx

```diff
@@ -248,3 +249,35 @@ TEST_CASE( "Lebedev-Laikov", "[1d-quad]" ) {
   test_fn(974);
 
 }
+
+TEST_CASE( "Ahrens-Beylkin", "[1d-quad]" ) {
+
+
+  auto test_fn = [&]( size_t nPts ) {
+
+    IntegratorXX::AhrensBeylkin<double> quad( nPts );
+
+    const auto& pts = quad.points();
+    const auto& wgt = quad.weights();
+
+    for( auto l = 1; l < 10; ++l )
+    for( auto m = 0; m <= l; ++m ) {
+
+      auto f = [=]( decltype(pts[0]) x ){
+        return spherical_harmonics(l,m,x[0],x[1],x[2]);
+      };
+
+      std::complex<double> res = 0.;
+      for( auto i = 0; i < quad.npts(); ++i )
+        res += wgt[i] * f(pts[i])* std::conj(f(pts[i]));
+
+      CHECK( 4.*M_PI*std::real(res) == Approx(1.) );
```

Comment:
I'm not sure where you got the nodes for this quadrature, but unlike the Lebedev-Laikov quadratures, the Ahrens-Beylkin weights already include the solid-angle integration factor (14*M_PI1). Removing this factor allows for the unit test to pass (confirmed locally)

Reply:
The rules are from Beylkin himself, it just took a year and running into him in person at a conference to get them :stuck_out_tongue_closed_eyes: 

Having to multiply the Lebedev weights afterwards by `4*M_PI` does not seem reasonable. I have modified the code so that that factor is included in the constructor.